sale_order_warn_message/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This module add a warning popup on sale to ensure warning is populated
+no only when partner is changed.
```

Comment:
```suggestion
not only when partner is changed.
```

Reply:
The description should be enhanced as it is not a popup but a permanent alert warning on sale order form.

An image of the form should be great too (optional).